config/global.ini.php

```diff
@@ -561,6 +561,10 @@
 ; on a per-request basis;
 pivot_by_filter_default_column_limit = 10
 
+; ONLY CHANGE THIS VALUE WHEN YOU DO NOT USE PIWIK ARCHIVING, SINCE THIS COULD CAUSE PARTIALLY MISSING ARCHIVE DATA
+; Whether to force a new visit at midnight for every visitor. Default 1.
+force_new_visit_midnight = 1
```

Comment:
maybe we rename to `create_new_visit_after_midnight` and add it after `create_new_visit_when_website_referrer_changes` ?